C_CPP/OOPs in C++/Inheritance and its types or Function overriding/multiple_inheritance_doc.md

```diff
@@ -0,0 +1,135 @@
+# Multiple inheritance
+
+Multiple inheritance is a type of inheritance in which a class can inherit from more than one class.
+
+Therefore, in multiple inheritance, the derived class can access the different data members and member functions of its base classes according to the defined access modes. The access modes are defined using access specifiers. There are three access specifiers that can be used in inheritance in C++, they are: private, protected, and public.
+
+The below image demonstrates a simple example of multiple inheritance :
+
+<p align="center"> <img src="https://user-images.githubusercontent.com/61552413/142762283-e4deada2-c86b-433a-9e54-8cb05ff99f31.png" height="350"> </p>
+
+Here, class D will be regarded as a child class or derived class, and classes A, B and C will be regarded as the parent classes or the base classes.
+
+A real-life example of multiple inheritance could be, class Bat inheriting its properties and characteristics from classes Mammal and wingedAnimal.
+
+## Example
+
+```cpp
+#include <iostream>
```

Comment:
@vartikavr this is a pretty basic example, give another better example along with this.